jrqs-app/src/java/ua/dp/isd/jrqs/SqlBuilder.java

```diff
@@ -69,7 +67,12 @@ public void addCommaSeparatedParameters(String commaSeparatedParameters) {
             return;
         }
 
-        sqlParameters = commaSeparatedParameters.toLowerCase().split(",");
+        String[] splitted = commaSeparatedParameters.toLowerCase().split(",");
+        if (splitted.length > MAX_PARAMETERS) {
+            throw new IllegalArgumentException("The number of maximum allowed arguments have been exceeded.");
```

Comment:
We support "expand range" feature that converts requests like FR1-100 into FR1, FR2, FR3 and so on.